lib/name_drop/configuration.rb

```diff
@@ -15,18 +15,4 @@ def initialize
       @access_token = 'NotARealAccessToken'
     end
   end
-
-  class << self
```

Comment:
I'm of the opinion that the file and the module should be tightly coupled, so you know where to look for everything related to that module.

Can roll this back though if there are strong opposing views?